docs/source/model_design/general_research.rst

```diff
@@ -679,11 +679,131 @@ Conducting Systematic Literature Searches/Reviews
 Directed Acyclic Graphs (Causal Diagrams)
 ------------------------------------------
 
-.. todo::
+.. important::
+
+  The figures represented in this section represent directed acyclic graphs/causal diagrams. These figures are distinct from compartamental model diagrams (such as SIR models). For our purposes we will represent directed acyclic graphs as *circles* connected by arrows and compartamental model diagrams as *squares* connected by arrows.
+
+  Additionally, for the purposes of directed acyclic graphs, variables will be labeled with capital letters (A, B, C, etc.) and arrows will be labeled with lower case letters (a, b, c, etc.).
+
+Introduction to Directed Acyclic Graphs
++++++++++++++++++++++++++++++++++++++++
+
+Directed acyclic graphs (DAGs), also termed causal diagrams, are composed of variables that are linked together by directional arrows. Critically, the solid arrows in DAGs represent a causal relationship between an exposure (start of the arrow) and outcome (node of the arrow), representing the *directional* aspect of DAGs. Notably, a directional arrow in a DAG does not offer any information about whether the exposure causes in increased or decreased probability of the outcome, only that there is some causal realtionship bewteen the exposure and outcome. 
+
+The *acyclic* nature of DAGs is that there cannot be feedback loops between any variables in the graph. In this way, the arrows in DAGs can be interpreted as the passage of time so that the exposure (cause) must always occur temporally *before* the outcome (effect). See the diagram below for an example of an inappropriate and appropriate DAG with regard to temporality.
+
+.. image:: acyclic_dags.svg
```

Comment:
This is really clear, i like this diagram

Reply:
I like it too, though perhaps it's not totally clear from the diagram that you also can't have feedback loops that involve more than two variables. But I think the interpretation in terms of flow of time helps clarify that even if we don't spell it out explicitly.